packages/backend/src/database/migrations/20240207133121_add-english-dictionary-with-custom-stop-words.ts

```diff
@@ -0,0 +1,35 @@
+import { Knex } from 'knex';
```

Comment:
you could merge into a single migration, why separate?

Reply:
good point, this was merged over multiple PRs which resulted in separate migrations.
also, merging them into 1 single file won't impact readability too much, doing it 💪